docs/plugins/discovery-ec2.asciidoc

```diff
@@ -124,6 +124,7 @@ The available values are:
 * `eu-central` (`eu-central-1`) for EU (Frankfurt)
 * `sa-east` (`sa-east-1`) for South America (São Paulo)
 * `cn-north` (`cn-north-1`) for China (Beijing)
+* `ca-central-1` (`ca-central-1`) for Canada (Central)
```

Comment:
Since its ca-central-1, in the spirit of following the convention that we can omit the `-1`, we should also support `ca-central` for `ca-central-1`, so this line should read:
```
`ca-central` (`ca-central-1`) for Canada (Central)
```